hearth/lib/hearth/http/client.rb

```diff
@@ -115,7 +114,8 @@ def configure_ssl(http)
       # @return [Net::HTTP::Request]
       def build_net_request(request)
         request_class = net_http_request_class(request)
-        req = request_class.new(request.url, request.headers.to_h)
+        # Trailers are not supported in Net::HTTP, but get all fields anyway.
```

Comment:
I think we should probably only get headers and then fail if there are any trails